source/patterns/@aws-solutions-constructs/aws-lambda-stepfunctions/README.md

```diff
@@ -0,0 +1,100 @@
+# aws-lambda-stepfunctions module
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
+
+> All classes are under active development and subject to non-backward compatible changes or removal in any
+> future version. These are not subject to the [Semantic Versioning](https://semver.org/) model.
+> This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
+---
+<!--END STABILITY BANNER-->
+
+| **Reference Documentation**:| <span style="font-weight: normal">https://docs.aws.amazon.com/solutions/latest/constructs/</span>|
+|:-------------|:-------------|
+<div style="height:8px"></div>
+
+| **Language**     | **Package**        |
+|:-------------|-----------------|
+|![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python|`aws_solutions_constructs.aws_lambda_stepfunctions`|
+|![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript|`@aws-solutions-constructs/aws-lambda-stepfunctions`|
+|![Java Logo](https://docs.aws.amazon.com/cdk/api/latest/img/java32.png) Java|`software.amazon.awsconstructs.services.lambdastepfunctions`|
+
+This AWS Solutions Construct implements an AWS Lambda function connected to an AWS Step Functions.
+
+Here is a minimal deployable pattern definition in Typescript:
+
+``` typescript
+import { LambdaToStepFunctions } from '@aws-solutions-constructs/aws-lambda-stepfunctions';
```

Comment:
Name needs to be LambdaToStepfunctions to match design guidelines (we only uppercase the first letter - while that seems odd here if you think of LambdaToSqs the reason for the pattern becomes clearer). 

(same for Props interface)